workspaces_test/delete_workspace_test.go

```diff
@@ -0,0 +1,28 @@
+package workspaces
+
+import (
+	"github.com/seatsio/seatsio-go/v11"
+	"github.com/seatsio/seatsio-go/v11/shared"
+	"github.com/seatsio/seatsio-go/v11/test_util"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestDeleteInactiveWorkspace(t *testing.T) {
+	t.Parallel()
+	company := test_util.CreateTestCompany(t)
+	client := seatsio.NewSeatsioClient(test_util.BaseUrl, company.Admin.SecretKey)
+
+	workspace, err := client.Workspaces.CreateProductionWorkspace(test_util.RequestContext(), "my workspace")
+	require.NoError(t, err)
+	require.True(t, workspace.IsActive)
```

Comment:
Don't think it's necessary to assert that the workspace is active - that's the default for all workspaces.